.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: go
+
+go:
+  - 1.11.x
+  - 1.12.x
+  - 1.13.x
+  - 1.14.x
+  - 1.15.x
+  - 1.16.x
+
+git:
+  depth: 1
+
+before_script:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.40
+
+script:
+  - golangci-lint run --modules-download-mode=vendor --deadline=5m
+  - go test -v -race ./...
```

Comment:
Please add a blank line at the end

Reply:
Need to make migration to Github Actions: https://docs.github.com/en/actions
TravisCI is closed for open source projects.

Reply:
done